bootstrap/src/main/kotlin/jp/co/soramitsu/notary/bootstrap/dto/Model.kt

```diff
@@ -0,0 +1,90 @@
+package jp.co.soramitsu.notary.bootstrap.dto
+
+import jp.co.soramitsu.iroha.java.TransactionBuilder
+import jp.co.soramitsu.notary.bootstrap.genesis.getIrohaPublicKeyFromHexString
+import java.security.KeyPair
+import javax.xml.bind.DatatypeConverter
+
+private interface DtoFactory<out T> {
+    fun getDTO(): T
+}
+
+open class Conflicatable(var errorCode: String? = null, var message: String? = null)
+
+data class BlockchainCreds(
+    val private: String? = null,
+    val public: String? = null,
+    val address: String? = null
+)
+
+data class IrohaAccountDto(
+    val title: String = "",
+    val domainId: String = "",
+    val creds: List<BlockchainCreds> = listOf()
+)
+
+data class IrohaAccount(val title: String, val domain: String, val keys: HashSet<KeyPair>) :
+    DtoFactory<IrohaAccountDto> {
+    override fun getDTO(): IrohaAccountDto {
+        val credsList: ArrayList<BlockchainCreds> = ArrayList()
+        keys.forEach {
+            credsList.add(
+                BlockchainCreds(
+                    DatatypeConverter.printHexBinary(it.private.encoded),
+                    DatatypeConverter.printHexBinary(it.public.encoded)
+                )
+            )
+        }
+        return IrohaAccountDto(this.title, domain, credsList)
+    }
+}
+
+data class Peer(val peerKey: String = "", val hostPort: String = "localhost:10001")
+data class Project(val project: String = "D3", val environment: String = "test")
+
+data class GenesisRequest(
+    val accounts: List<IrohaAccountDto> = listOf(),
```

Comment:
Seems more neatly```val accounts = emptyList<IrohaAccountDto>()```

Reply:
fixed but with type annotation because it is required